modules/serverless-python-function-return-values.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-python-function-return-values_{context}"]
+= Python function return values
+
+Functions may return any value supported by https://flask.palletsprojects.com/en/1.1.x/quickstart/#about-responses[Flask] because the invocation framework proxies these values directly to the Flask server.
+
+.Example
+[source,python]
+----
+def main(context: Context):
+    body = { "message": "Howdy!" }
+    headers = { "content-type": "application/json" }
+    return body, 200, headers
+----
+
+Functions can set both headers and response codes as secondary and tertiary response values from function invocation.
+
+== Returning CloudEvents
+
+Developers can use the `@event` decorator to tell the invoker that the function return value must be converted to a CloudEvent before sending the response.
+
+.Example
+[source,python]
+----
+@event("event_source"="/my/function", "event_type"="my.type")
+def main(context):
+    # business logic here
+    data = do_something()
+    # more data processing
+    return data
+----
+
+This example sends a CloudEvent as the response value, with a type of `"my.type"`, a source of `"/my/function"`, and the data property set to `data`. The `event_source` and `event_type` decorator attributes are both optional.
+
+If not specified, the CloudEvent `event_source` attribute is set to `"/parliament/function"`, and the `event_type` attribute is set to `"parliament.response"`.
```

Comment:
@lance can you provide explanations for the `parliament` ones cos I'm not sure what they do and it's not explained [here](https://github.com/boson-project/func/blob/main/docs/guides/python.md#cloudevents) - thanks!

Reply:
@abrennan89 the `event_source` and `event_type` attributes refer to properties in a Python CloudEvent, as manifested by [cloudevents/sdk-python](https://github.com/cloudevents/sdk-python). These properties are required per the CloudEvents spec. For convenience, since the user may not care about the type and/or source values, we populate them with default values. It's hard to get into a lot of detail on all of this because we are simply referencing external CNCF libraries and specifications.

Reply:
Above comment removes mention of them anyway, probably no need to mention if it's just a default setting and we're linking to the CloudEvent spec